toolchains/lowrisc_rv32imcb/repository.bzl

```diff
@@ -8,8 +8,8 @@ def lowrisc_rv32imcb_repos(local = None):
     http_archive_or_local(
         name = "lowrisc_rv32imcb_files",
         local = local,
-        url = "https://github.com/lowRISC/lowrisc-toolchains/releases/download/20230519-1/lowrisc-toolchain-rv32imcb-20230519-1.tar.xz",
-        sha256 = "08b1ba2089aa4206efdca93b7ab70152c3fe16ef4c6ee112a4f35ee3dc65aa8c",
-        strip_prefix = "lowrisc-toolchain-rv32imcb-20230519-1",
+        url = "https://github.com/lowRISC/lowrisc-toolchains/releases/download/20240206-1/lowrisc-toolchain-rv32imcb-20240206-1.tar.xz",
```

Comment:
maybe we should have a variable `version` instead of copying the version three times in three different places?